generators/init/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint no-underscore-dangle: 0 */
+
+/*
+* License-Identifier: Apache-2.0
+*/
+const Generator = require('yeoman-generator');
+const simpleFabricaConfigJson = require('../../samples/fabricaConfig-1org-1channel-1chaincode');
+
+module.exports = class extends Generator {
+  async copySimpleFile() {
+    this.fs.writeJSON(
+      'fabrica-network-config.json',
+      simpleFabricaConfigJson,
+    );
```

Comment:
rozumiem, że na razie żadnego chaincoda nie dajemy

Reply:
tak - i nie sądzę żebyśmy kiedokolwiek dawali. imo to powinien być kompletnie oddzielny projekt, który nie będzie specjalnie atrakcyjny. bo sa już próby robienia tego. np. tu:
https://github.com/IBM-Blockchain/generator-fabric

Reply:
no moim zdaniem to byłby fajny feature, bo wtedy od razu masz działającą sieć, na której możesz coś zrobić. ale do przegadania na przyszłość. a tak musisz wiedzieć, że takie narzędzie jest, znaleźć je i użyć (i mieć yeomana na dysku). może kiedyś fajną opcją by było zaraz po init zapytać, czy chcesz stworzyć sieć i wtedy odpalić ten ich generator.

Reply:
stanęło na oddzielnym tickecie pod to